technology/core-skills/tdd/inappropriate.md

```diff
@@ -1,11 +1,24 @@
 # Inappropriate TDD
```

Comment:
I wonder if we could change the title to something like "When could TDD be inappropriate" or something similar. When I read the title, I expected to read examples of bad TDD, but this is guidance on when not to use it.